src/modules/ekf2/test/test_EKF_mag.cpp

```diff
@@ -78,7 +78,7 @@ TEST_F(EkfMagTest, fusionStartWithReset)
 	_sensor_simulator.runSeconds(_init_duration_s);
 
 	// THEN: the fusion initializes using the mag data and runs normally
-	EXPECT_NEAR(_ekf_wrapper.getYawAngle(), mag_heading, radians(0.1f));
+	EXPECT_NEAR(_ekf_wrapper.getYawAngle(), mag_heading, radians(1.f));
```

Comment:
This is due to the sensor simulator sending in default mag initially before the mag test sets the desired data.